apps/editing-toolkit/editing-toolkit-plugin/global-styles/class-global-styles-fonts-message-control.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * A WP_Customize_Control for giving instructions on how to edit your fonts from within the Global Styles plugin.
+ *
+ * @package Automattic\Jetpack\Global_Styles
+ */
+
+namespace Automattic\Jetpack\Global_Styles;
+
+/**
+ * Class Global_Styles_Fonts_Message_Control
+ */
+class Global_Styles_Fonts_Message_Control extends \WP_Customize_Control {
+
+	/**
+	 * Creates a link to the page editor for the user's homepage with the global styles sidebar opened.
+	 */
+	private function get_link_to_editor_with_global_styles_sidebar() {
+		$homepage_id = \get_option( 'page_on_front' );
+		if ( empty( $homepage_id ) ) {
+			return null;
+		}
+
+		$base_url = null;
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		if ( 'http://calypso.localhost:3000' === $_GET['calypsoOrigin'] ) {
+			$base_url = 'http://calypso.localhost:3000/';
+		} else {
+			$base_url = 'https://www.wordpress.com/';
+		}
```

Comment:
Don't forget that we have horizon.wordpress.com, and wpcalypso.wordpress.com, as well.

I think you can use `WPCOM_Masterbar::get_calypso_site_slug`, but it would need an `exists` check with a fallback, since it's only available on wpcom. For example: https://github.com/Automattic/jetpack/blob/529d07f441bb4667acd773918563296878e19cb4/projects/packages/status/src/class-status.php#L268

Reply:
@creativecoder @kwight With regard to horizon.wordpress.com and wpcalypso.wordpress.com,

Is this something that we can follow up with right after deployment?

If we leave it out for right now, it just means if someone clicks this block editor link, they will be directed to the wordpress.com version.

<img src="https://user-images.githubusercontent.com/10274366/124938291-2ca5f080-dfd6-11eb-9496-49e659df5d70.png" width=250 />

Reply:
@DanielSmithMichigan Let's do the `WPCOM_Masterbar::get_calypso_site_slug` fix now, as well as verifying Tracks are firing before merging. It's very close!

Reply:
@kwight Yep I just pushed the WPCOM_Masterbar::get_calypso_site_slug fix.

The tracks events are firing as well.

Reply:
Just waiting for the atomic artifact to finish building. And I'll try atomic one more time to double check against my most recent changes. Then we should be good to go.

Reply:
Everything is working on atomic. I don't have anything on my plate for this PR at this moment.